ext/json/php_json_parser.h

```diff
@@ -81,4 +81,6 @@ PHP_JSON_API int php_json_parse(php_json_parser *parser);
 
 int php_json_yyparse(php_json_parser *parser);
 
+PHP_JSON_API const php_json_parser_methods get_validate_methods();
```

Comment:
I don't think it should be part of the API so better remove `PHP_JSON_API`

Reply:
done